src/app/reducers/media-wall-pagination.ts

```diff
@@ -0,0 +1,68 @@
+import { createSelector } from 'reselect';
+import * as wallPaginationAction from '../actions/media-wall-pagination';
+
+/**
+ * Each reducer module must import the local `State` which it controls.
+ *
+ * Here the `State` contains three properties:
+ * @prop [entities: ApiResponseResult[]] array of response items returned by the api.
+ * @prop [valid: boolean] shows the validity of data present in state.
+ * 												Used to detect wheather the loading has been success/fail.
+ */
+export interface State {
+	wallPaginating: boolean;
+}
+
+/**
+ * There is always a need of initial state to be passed onto the store.
+ *
+ * @prop: metadata: null
+ * @prop: entities: []
+ * @prop: hashtags: []
+ * @prop: aggregations: null
+ * @prop: valid: true
+ */
+export const initialState: State = {
+	wallPaginating: false
+};
+
+
+/**
+ * The actual reducer function. Reducers can be thought of as the tables in the DataBase.
+ * These are the functions which are responsible for maintaing, and updating the
+ * `State` of the application they control.
+ *
+ * Here the reducer controls the part of state which is responsilble for storing the
+ * results fetched from the API.
+ */
+export function reducer(state: State = initialState, action: wallPaginationAction.Actions): State {
```

Comment:
It appears that this file and `src/app/reducers/media-wall-search.ts` can be merged and this method can also be merged.

Could you see if these can b merged :)